README.md

```diff
@@ -26,6 +26,20 @@ tools do not aim to provide an optimization platform or a higher-level compiler
 target; instead they aim for full fidelity and compliance with the spec (e.g.
 1:1 round-trips with no changes to instructions).
 
+## Installation
+You can install Wabt command line tools on Linux or macOS using Homebrew:
+```sh
+brew install wabt
+```
+
+On Linux, you can also use a package manager like `apt`:
+```sh
+sudo apt install wabt
+```
+Windows or otherwise interested users can download installers and
```

Comment:
You about: "You can also download prebuilt binaryes for many platforms directly from the github releases page".

Reply:
Thank you